pkg/trace/config/apply.go

```diff
@@ -173,9 +173,23 @@ func (c *AgentConfig) applyDatadogConfig() error {
 	if config.Datadog.IsSet("apm_config.log_file") {
 		c.LogFilePath = config.Datadog.GetString("apm_config.log_file")
 	}
+
```

Comment:
APM styling nit, but can we remove the empty lines before and after the `if` block? I find that generally the curly braces provide enough spacing, and I've also noticed that in Go spacing like this to create "paragraphs of code" is not really used much looking at larger projects and the `golang/go` repository itself.